### PR TITLE
add live test script

### PR DESCRIPTION
**Pull request by trust2065**
add live test script
```
    "testcafe": "testcafe 'chrome:headless' test/e2e-test/*.js",
    "testcafe:chrome": "testcafe 'chrome' test/e2e-test/*.js",
    "testcafe:chrome-headless": "testcafe 'chrome:headless' test/e2e-test/*.js",
    "testcafe-live": "testcafe-live 'chrome:headless' test/e2e-test/*.js",
    "testcafe-live:chrome": "testcafe-live 'chrome' test/e2e-test/*.js",
    "testcafe-live:chrome-headless": "testcafe-live:headless 'chrome' test/e2e-test/*.js",
```

change your focus on terminal then you can
ctrl+s - stop current test run;
ctrl+r - restart current test run;
ctrl+w - turn off/on files watching;
ctrl+c - close opened browsers and terminate the process.

### CODE REVIEW

**Comment by trust2065** [comment]
@josephj 
Would you please review again?

**Comment by trust2065** [comment]
@josephj Would you please review again?

File: package.json [hunk 1]
```diff
@@ -61,7 +62,12 @@
     "lint": "eslint .",
     "test": "cross-env NODE_ENV=test mocha test/unit-test test/integration-test",
     "test:frontend": "cd client && yarn test",
-    "test:functional": "node ./test/e2e-test/server.js ",
+    "testcafe": "testcafe 'chrome:headless' test/e2e-test/*.js",
```
Comment (josephj) [inline package.json]:
```
"testcafe": "npm run testcafe:chrome-headless",
```

File: package.json [hunk 2]
```diff
@@ -61,7 +62,12 @@
...
+    "testcafe:chrome": "testcafe 'chrome' test/e2e-test/*.js",
+    "testcafe:chrome-headless": "testcafe 'chrome:headless' test/e2e-test/*.js",
+    "testcafe-live": "testcafe-live 'chrome:headless' test/e2e-test/*.js",
```
Comment (josephj) [inline package.json]:
```
"testcafe-live": "npm run testcafe-live:chrome-headless",
```

**Pull request merged**